src/constants/MarkdownFiles/posts/2025-09-12-gsoc-25-bishoy-wadea-final-report.md

```diff
@@ -0,0 +1,289 @@
+---
+title: "GSoC’25 Final Report by Bishoy Wadea"
+excerpt: "Developing 10 Math Sugar Activities"
+category: "DEVELOPER NEWS"
+date: "2025-09-12"
+slug: "2025-09-12-gsoc-25-bishoy-wadea-final-report"
+author: "@/constants/MarkdownFiles/authors/bishoy-wadea.md"
+tags: "gsoc25,sugarlabs,Final-Report,BishoyWadea"
+image: "assets/Images/GSOC.webp"
+---
+
+# Google Summer of Code 2025: Educational Activities for Sugar
+
+**Project:** Developing 10 Math Sugar Activities  
+**Mentors:** [Ibiam Chihurumnaya](https://github.com/chimosky)  
+**Assisting Mentor:** [Walter Bender](https://github.com/walterbender/)  
+**Reporting Period:** 2025-05-08 – 2025-09-12  
+
+---
+
+During Google Summer of Code 2025, I embarked on an ambitious journey to develop ten innovative educational activities for the Sugar learning platform. My mission was to create engaging, educational experiences that would enhance various cognitive skills in young learners through interactive gameplay and cutting-edge technologies. Throughout this project, I focused on implementing diverse educational concepts ranging from classic puzzles to AI-powered learning tools, ensuring each activity not only served its educational purpose but also captivated users with intuitive interfaces and engaging mechanics.
+
+This project allowed me to explore various domains including game development, 3D graphics, network programming, and machine learning, while contributing meaningfully to the Sugar community. Through challenges and triumphs, I successfully delivered a comprehensive suite of activities that span mathematical reasoning, spatial intelligence, pattern recognition, and artificial intelligence education.
+
+## Table of Activities
+
+| **Activity Name**                        | **Link**                                                                                                                                       |
+|------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------|
+| **Four Color Map Puzzle**                | [![Icon](https://raw.githubusercontent.com/Bishoywadea/Four-Color-Map/e7988b391e2717360437b3e06492d329493b7cf6/activity/activity-fourcolormap.svg)](https://github.com/Bishoywadea/Four-Color-Map)                                          |
```

Comment:
Please either remove the word "Icon" or write a proper caption for each of these images. They are displaying under the images.

That said, adding the word "icon" as alt text would be nice. However, what you have here is rendering as a caption, not alt text.